content/e/mcw/2018-12-08-mcw-show-1.md

```diff
@@ -6,18 +6,22 @@ authors = ["M3n747"]
 chronology = ["mcw"]
 [extra]
 city = "Siemianowice Śląskie"
+[extra.gallery.1]
+path = "2018-12-08-mcw-show-plakat.jpg"
+caption = "Official poster. Pictured from left to right are: Diana, [Johny Vegas](@/w/johny-vegas.md), [RAV](@/w/rav.md)."
+source = "Johny Vegas"
 +++
 
-In December 2018 MCW began cooperation with Sekcja Sportów Siłowych (_Stength Sports Section_) from Siemianowice Śląskie, thus getting an opportunity to put together a show in front of around 200 people. The event took place around Sait Nicholas Day, which in Poland always falls on the 6th of December - the exact date might of the show might differ slightly. The event was held in a gym hall of what appears to be a local school.
+In December 2018 MCW began cooperation with Sekcja Sportów Siłowych (_Stength Sports Section_) from Siemianowice Śląskie, thus getting an opportunity to put together a show in front of around 200 people. The event was held in a gym hall of what appears to be a local school.
```

Comment:
Typo: strength, also the common term is "sports athletics" but that doesn't get us the cool alliteration matching the Polish name.